tests/sentry/hybrid_cloud/rpc_services/test_control_organization_provisioning.py

```diff
@@ -98,6 +105,17 @@ def test_organization_provisioning_happy_path(self):
             rpc_org_slug=rpc_org_slug, user_id=self.provision_user.id
         )
 
+    def test_organization_provisioning_before_user_provisioning(self):
+        provisioning_options = self.generate_provisioning_args(
+            name="sentry", slug="sentry", email="test-owner@sentry.io", default_team=True
+        )
+        slug = control_organization_provisioning_rpc_service.provision_organization(
+            region_name="us", org_provision_args=provisioning_options
+        )
+        self.assert_slug_reservation_and_org_exist(
+            rpc_org_slug=slug,
```

Comment:
Should we do an org member ownership check here? I think I saw the getsentry-side had assertions for this already but it may be good to add them here as well.